### ISSUE TITLE
Cannot run JNLP file on Windows with space in profile directory

### ISSUE THREAD

**Issue by mhoare1984** [AdoptOpenJDK/IcedTea-Web#619]
I am unable to run any JNLP file in Windows with AdoptOpenJDK/IcedTea-Web for user accounts that have a space in the account name. E.g. The windows profile is C:\Users\X[SPACE]Y\. If I create a user account on the same machine without a space, I can launch any JNLP file without error. Due to odd ball software requirements we have a few accounts that require spaces.

Below is the Java console output.

```
net.sourceforge.jnlp.LaunchException: Fatal: Launch Error: Could not launch JNLP file. The application has not been initialized, for more information execute javaws/browser from the command line and send a bug report.
     at net.sourceforge.jnlp.Launcher.launchApplication(Launcher.java:582)
     at net.sourceforge.jnlp.Launcher$TgThread.run(Launcher.java:945)
 Caused by: java.lang.ClassNotFoundException: Wallpaper
     at net.sourceforge.jnlp.runtime.JNLPClassLoader.loadClass(JNLPClassLoader.java:1644)
     at net.sourceforge.jnlp.Launcher.launchApplication(Launcher.java:556)
     ... 1 more
netx: Launch Error: Could not launch JNLP file. (Wallpaper)
java.lang.ClassNotFoundException: Wallpaper
     at net.sourceforge.jnlp.runtime.JNLPClassLoader.loadClassExt(JNLPClassLoader.java:1802)
     at net.sourceforge.jnlp.runtime.JNLPClassLoader.loadClass(JNLPClassLoader.java:1602)
     at net.sourceforge.jnlp.Launcher.launchApplication(Launcher.java:556)
     at net.sourceforge.jnlp.Launcher$TgThread.run(Launcher.java:945)
Starting application [Wallpaper] ...
```

**Comment by sclassen** [comment]
Which version are you running?

**Comment by mhoare1984** [comment]
1.8.3 bundled with the latest version of AdoptOpenJDK 1.8.0_242.

I found a workaround. Using itweb-settings.exe on the Cache settings section, change the “Select the location where temporary files are kept” to a directory that does not contain a space.

Then it works.

**Comment by sclassen** [comment]
OK, this is already fixed in the 2.0.0 branch
Check out https://openwebstart.com for a preview how ITW 2.0.0 will work
